test/DynamoCoreWpfTests/DynamoViewTests.cs

```diff
@@ -123,5 +124,36 @@ public void OpeningWorkspaceWithTrustWarning()
             Assert.IsFalse(ViewModel.FileTrustViewModel.ShowWarningPopup);
             DynamoModel.IsTestMode = true;
         }
+
+        [Test]
+        public void TestToastNotificationClosingBehavior()
+        {
+            var preferencesWindow = new PreferencesView(View);
+            preferencesWindow.Show();
+            DispatcherUtil.DoEvents();
+            string selectedLanguage = (string)((ComboBox)preferencesWindow.FindName("LanguageCmb")).SelectedItem;
+
+            ViewModel.PreferencesViewModel.SelectedLanguage = selectedLanguage == "English" ? "Español" : "English";
+
+            ViewModel.HomeSpace.HasUnsavedChanges = false;
+            if (View.IsLoaded)
+                View.Close();
+
+            if (ViewModel != null)
+            {
+                var shutdownParams = new DynamoViewModel.ShutdownParams(
+                    shutdownHost: false, allowCancellation: false);
+
+                ViewModel.PerformShutdownSequence(shutdownParams);
+            }
+
+            bool isToastNotificationVisible = (bool)(ViewModel.MainGuideManager?.ExitTourPopupIsVisible);
+            Assert.IsFalse(isToastNotificationVisible);
+        }
+
+        /// <summary>
+        /// Overrides the Exit base class method to custom handling
+        /// </summary>
```

Comment:
the summary is below the function here..

Reply:
Done.